src/java/org/apache/cassandra/db/Directories.java

```diff
@@ -349,6 +349,16 @@ private static boolean shouldEnableNode(double threshold, double currUtilization
         return underThreshold && nodeCanBeReEnabled && zeroNonTransientErrors && onlyExceededDiskThresholdTransientError;
     }
 
+    /**
+     * Finds busiest data directory and returns its utilization as a percentage.
+     *
+     * @return Map.Entry<DataDirectory, Double> where the key represents the directory
+     *         and the value represents percentage utilization of the directory.
+     */
+    public static Map.Entry<DataDirectory, Double> getMostUtilizedDataDir() {
+        return getMaxPathToUtilization();
+    }
+
     private static Map.Entry<DataDirectory, Double> getMaxPathToUtilization()
     {
```

Comment:
nit: Is there a reason not to rename `getMaxPathToUtilization()` to `getMostUtilizedDataDir` and make it public?

Reply:
It will make merging new releases of oss C* harder, but seen as 2.2.x is EOL it makes sense to do this. I'll keep the function name as is (getMaxPathToUtilization) but will directly make it public rather than creating a public version of the function with a different name.